### ISSUE TITLE
Check if command 'Usage:' was found

### ISSUE THREAD

**Issue by hatted** [tachtler/dovecot-backup#13]
I got an error in 1.11 for my Ubuntu1804.
Check if command '/bin/mktemp' was found ..................[ OK ]
Check if command 'Usage:' was found .......................[FAILED]

I find the error and removed the --skip-alias to make it works.
MV_COMMAND=`which mv`
#MV_COMMAND=`which --skip-alias mv`

**Comment by tachtler** [comment]
Hi hatted,

I have tested it on an Ubuntu 18.04 LTS (virtual machine) with the following result
```
klaus@ubuntu:~$ which --skip-alias mv
Illegal option --
Usage: /usr/bin/which [-a] args
```
```
klaus@ubuntu:~$ command which mv
/bin/mv
```
Your problem has been solved with the **new version 1.12**!

@hatted: **_Please, can you confirm that?_**

For compatibility reasons with other Linux distributions, I've made the following small changes:
`which --skip-alias mv`
to
`command which mv`

Thank you very much for your support!
Klaus.

**Comment by hatted** [comment]
Yes, you are correct.   This can fix the error in Ubuntu1804.   or I tried using
`which mv --skip-alias`
also works if you need the skip-alias parameters.

Now I found another issue.   I close this issue and open another one.
